trello.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pprint
+import requests
+
+pp = pprint.PrettyPrinter(indent=4)
+app_board_name='To Do App'
+trello_member='brendanmcdaniel'
+trello_endpoint='https://api.trello.com/1/'
+
+class item:
```

Comment:
By convention Python class names are written in PascalCase.

Also they usually belong in their own file. It definitely does in this case because this class is not specific to Trello interaction - it is your app's definition of an item.